utils/build_locally.sh

```
@@ -22,8 +22,14 @@ case $choice in
     docker run --rm -e MAKEFLAGS -e OPENTURNS_NUM_THREADS=2 -e OMP_NUM_THREADS=1 -v `pwd`:/io openturns/archlinux-python /io/.ci_support/run_docker_coverage.sh `id -u`:`id -g`
     ;;
   "4" | "aarch64")
-    docker pull openturns/debian-aarch64
-    docker run --rm -e MAKEFLAGS -e OPENTURNS_NUM_THREADS=2 -v `pwd`:/io openturns/debian-aarch64 /io/.ci_support/run_docker_aarch64.sh
+    if test `uname -m` == "aarch64"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)